TESTING.MD

```diff
@@ -0,0 +1,96 @@
+---
+weight: 121
+toc: true
+title: Tests during development
+menu:
+    docs:
+        parent: contributing
+lead: ""
+lastmod: "2023-09-10T08:48:57+00:00"
+images: []
+draft: false
+description: Understanding and running tests when developing Prometheus-Operator
+date: "2023-09-10T00:00:00+00:00"
+---
+
+When contributing code to Prometheus-Operator, you'll notice that every Pull Request will run against an extensive test suite. Among an extensive list of benefits that tests brings to the Project's overall health and reliability, it can be the reviewer's and contributors's best friend during development:
+
+* Test cases serve as documentation, providing insights into the expected behavior of the software.
+* Testing helps prevent regressions by verifying that new changes don't break existing functionality.
+* Running tests locally accelerate the feedback loop, removing the dependency that contributors might have on CI when working on a Pull Request.
+
+This document will focus on teaching you about the different test suites that we currently have and how to run different scenarios to help you development experience!
+
+# Test suites
+
+## Unit tests
+
+> Explain different unit test suites
+
+> test short
+> test long
```

Comment:
(nit) I always run all the tests and never used the short/long distinction. Not sure if it's useful to keep it.